src/snapshot_middleware/project.rs

```diff
@@ -30,6 +30,11 @@ pub fn snapshot_project(
     });
 
     context.add_path_ignore_rules(rules);
+    context.set_emit_legacy_scripts(
+        project
+            .emit_legacy_scripts
+            .unwrap_or_else(|| emit_legacy_scripts_default().unwrap()),
```

Comment:
```suggestion
        project
            .emit_legacy_scripts
            .or_else(emit_legacy_scripts_default).unwrap(),
```
This is more idiomatic and avoids the closure + double unwrapping